<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_marker_editor</class>
 <widget class="QFrame" name="wgt_marker_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item>
    <widget class="QWidget" name="wgtBWCF" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblCF">
           <property name="text">
            <string>Center Freq (Mhz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leCF"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgtBW" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblBW">
           <property name="text">
            <string>Bandwidth (Mhz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leBW"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblLowF">
           <property name="text">
            <string>Low Freq (Mhz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLowF"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblHighF">
           <property name="text">
            <string>High Freq (Mhz)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leHighF"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_st_end_t" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="wgtstatime" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblStartTime">
           <property name="text">
            <string>Start Time (S)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leStartTime"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblEndTime">
           <property name="text">
            <string>End Time (S)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leEndTime"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_11" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblDuration">
        <property name="text">
         <string>Duration (S)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtTimeChks" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkNoStartT">
        <property name="text">
         <string>No Start Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkNoEndT">
        <property name="text">
         <string>No End Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtTags" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTags">
        <property name="text">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTags"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
